File: fundamentals/palindrome_number/palindrome_number.py
def palindrome_number(n):
    reversed = 0
    original = n
    
    if n == 0: return True
    elif n < 0: return False
    else:
        while n > 0:
            num = n % 10
            reversed = reversed * 10 + num
            n = n // 10
        if reversed == original: return True
        else: return False
        
n = 1234321
print("Is the number a palindrome: ", palindrome_number(n))

    

File: fundamentals/palindrome_number/palindrome_number.ts
export {};

function palindroneNumber(n: number): boolean {
  const original = n;
  let reversed = 0;

  if (n === 0) return true;
  else if (n < 0) return false;
  else {
    while (n > 0) {
      const num = n % 10;
      reversed = reversed * 10 + num;
      n = Math.floor(n / 10);
    }
    if (reversed === original) return true;
    else return false;
  }
}

const n = 121;
console.log("Is the number a palindrome: ", palindroneNumber(n));
